routing/routing_session.hpp

```diff
@@ -105,6 +106,15 @@ class RoutingSession
   inline void SetState(State state) { m_state = state; }
 
   Route const & GetRoute() const { return m_route; }
+  /// \returns true if any altitude information along |m_route| is available and
```

Comment:
Just altitude information. It's valid only if all the information is available. If not, there no way to interpret it correctly.